src/NativeScript/Metadata/Metadata.mm

```diff
@@ -112,6 +121,48 @@ static int compareIdentifiers(const char* nullTerminated, const char* notNullTer
     return nullptr;
 }
 
+const std::vector<const MethodMeta*> BaseClassMeta::members(const char* identifier, size_t length, MemberType type, bool includeProtocols, bool onlyIfAvailable) const {
+    ASSERT(type == MemberType::InstanceMethod || type == MemberType::StaticMethod);
+    const ArrayOfPtrTo<MemberMeta>* members;
+    switch (type) {
+    case MemberType::InstanceMethod:
+        members = &this->instanceMethods->castTo<PtrTo<MemberMeta>>();
+        break;
+    case MemberType::StaticMethod:
+        members = &this->staticMethods->castTo<PtrTo<MemberMeta>>();
+        break;
+    default:
+        members = nullptr;
+        ASSERT_NOT_REACHED();
+    }
+
+    std::vector<const MethodMeta*> result;
+    if (members != nullptr) {
+
+        for (int i = 0; i < members->count; i++) {
```

Comment:
Use `binarySearch` for finding a member with the requested name and then linearly include all others surrounding it. You can implement this logic in a new `binarySearchRange` function in `Array`.